### ISSUE TITLE
Binaries?

### ISSUE THREAD

**Issue by ramapcsx2** [simias/psx-hardware-tests#1]
Hi,
is there any chance you could upload the tests binaries?

Would be very much appreciated :)

**Comment by simias** [comment]
I want to do that but currently it's kind of a bunch of programs which output their results in various ways (mostly through the serial link which is not always easily accessible, even in emulators) so it wouldn't work so well as a test suite.

I'm actually trying to write a small "terminal" library (in the term branch) what would be used to display the results on the screen directly. Then I could use that to make something looking like a proper test suite. Progress is slow because I'm lazy and easily distracted but I'll get there eventually.

**Comment by ramapcsx2** [comment]
A whole suite would be great!

But if it's just about the SIO link, things are easier now, thanks to the huge Arduino movement :)

**Comment by simias** [comment]
Yeah having cheap and standard adapters for that stuff would be great, I use an Xplorer module flashed with a custom firmware but they haven't been manufactured since the early 2000 and it's closed source all the way down.

I documented my setup here: https://github.com/simias/rustation/wiki/Xplorer-FX-setup

**Comment by yanmingsohu** [comment]
This test will rescue the development of the simulator.
